data/sql/updates/world/2016_08_19_00.sql

```diff
@@ -0,0 +1,35 @@
+
+ALTER TABLE world_db_version CHANGE COLUMN 2016_08_14_02 2016_08_19_00 BIT;
+
+/*
+2016-08-16 00:44:35 ERROR: Table 'creature_loot_template' entry 1 isn't creature entry and not referenced from loot, 
+and then useless.
+
+Removed, items inside are lootable from no one mobs.
+*/
+DELETE FROM creature_loot_template WHERE entry = 1;
+DELETE FROM conditions WHERE SourceTypeOrReferenceId = 1 AND SourceGroup = 1;
+
+/*
+2016-08-16 00:44:35 ERROR: Table 'skinning_loot_template' entry 80102 isn't creature skinning id and not referenced from loot, 
+and then useless.
+
+Adjust values of entry and loot % for the real NPC
+*/
+DELETE FROM skinning_loot_template WHERE entry IN (80102,18343);
```

Comment:
I guess this is wrong, 80102 should be the skinloot for 18343 abd 29268 (according to TDB)